exercises/practice/react/.meta/example.ex

```diff
@@ -0,0 +1,164 @@
+defmodule React do
+  use GenServer
+
+  defmodule InputCell do
+    defstruct [:name, :value]
+    @type t :: %InputCell{name: String.t(), value: any}
+  end
+
+  defmodule OutputCell do
+    defstruct [:name, :inputs, :compute, :value, callbacks: []]
+
+    @type t :: %OutputCell{
+            name: String.t(),
```

Comment:
Do you think cell names should be atoms instead?

Reply:
> I chose to use cell names as strings instead of atoms, because I know that atoms are not garbage collected and this felt like the kind of system where you would programmatically create new cells, even though it doesn't happen here. But atoms are cleaner, so I'm on the fence.

I agree with the concern, let's keep strings. Maybe you could describe this in `.meta/design.md`? for future maintainer generations :)

Reply:
That's a good suggestion, I will do that. Those notes are only meant for maintainers right?

Reply:
Yes